### ISSUE TITLE
Change the implementation of link tags to use the <a> tag. 

### ISSUE THREAD

**Issue by rsg73626** [ecureuill/SELIProject#73]
Some link elements are not using the appropriate HTML5 tag. It creates usability and accessibility problems. For example, these links are not styled as a link when not focused (usability) and they aren't reachable by keyboard (accessibility).

## Expected Behavior

The links must have a different appearance from the rest of simple texts. 
Keyboard users also must be able to select every available link in all pages. 

## Current Behavior

The links are not reachable by keyboard.
The links are not styled as a link when not focused.

## Steps to Reproduce

**[SELI -  Accessibility test](https://docs.google.com/presentation/d/1OBRtT9S3NZWK2bq9wvEzXLxcoQMxMC_CesAsUgqkUww/edit#slide=id.g5f43f92aa5180d90_20) presentation (slides 19-20)**

## Environment

Browser: all 

## Possible Solution

The best, fastest and simplest solution for that is to use the native HTML5 tag for links.
